team-config.yml

```diff
@@ -125,4 +125,12 @@ lgy-iac:
     contact_channel: "#dts-legacy-apps-contact"
     build_notices_channel: "#dts-legacy-apps-azure-builds"
   tags:
-    application: IAC-Fees
\ No newline at end of file
+    application: IAC-Fees
+makeaplea:
+  team: "Child Arrangement Service"
```

Comment:
is this the right team name?

Reply:
@timja oops, I did copy from C100 and missed it to change the team. I did change it to appropriate team name.